components/SightPreview.js

```diff
@@ -1,39 +1,47 @@
 import styled from "styled-components";
 import Image from "next/image";
+import DetailsPopUp from "./DetailsPopUp";
+import { useState } from "react";
 
 export default function TripPreview({ sight }) {
+  const [popUp, setPopUp] = useState(false);
+
+  function handlePopUp() {
+    setPopUp(!popUp);
+  }
+  console.log(popUp);
```

Comment:
In my recent reviews from coaches it was recommended to delete not needed console.logs.